src/Sylius/Bundle/CoreBundle/Form/DataTransformer/ChannelAndCurrencyPricingConfigurationTransformer.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\CoreBundle\Form\DataTransformer;
+
+use Symfony\Component\Form\DataTransformerInterface;
+use Webmozart\Assert\Assert;
+
+/**
+ * @author Arkadiusz Krakowiak <arkadiusz.krakowiak@lakion.com>
+ */
+final class ChannelAndCurrencyPricingConfigurationTransformer implements DataTransformerInterface
```

Comment:
IMO it should be a part of `ChannelAndCurrencyBasedConfigurationType`, it's not gonna be used anywhere else.